src/commons/TestUtils.ts

```diff
@@ -27,4 +30,24 @@ export namespace TestUtils {
   export function extractNumbersFromString(str: string): string {
     return str.replace(/[^0-9]/g, '');
   }
+
+  /**
+   * Return generic type of data by provided file path
+   * path File located in wdio config file
+   * example of using provided in spec src/test/specs/GetTestDataFileSpec.ts
+   * @param dataTag type of file
+   *  configDataFilePath will take from wdio.config file
+   */
+  export function getData<T>(dataTag: string = process.env.TEST_DATA_TAG): T {
+    // @ts-ignore
+    const dataFilename = browser.config.configDataFilePath;
+    const dataFilePath = path.resolve(process.cwd(), dataFilename);
+    if (dataFilename === undefined) {
```

Comment:
We can do this step before generating the `dataFilePath`

Reply:
added